src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4encoder/dropdownlist/VideoInputDropdown.java

```diff
@@ -0,0 +1,92 @@
+package com.avispl.symphony.dal.avdevices.encoderdecoder.haivision.x4encoder.dropdownlist;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * VideoInputDropdown class defined the enum for monitoring and controlling process
+ *
+ * @author Ivan / Symphony Dev Team<br>
+ * Created on 3/8/2022
+ * @since 1.0.0
+ */
+public enum VideoInputDropdown {
+
+    BNC_1("BNC_1", 0),
+    BNC_2("BNC_2", 1),
+    BNC_3("BNC_3", 2),
+    BNC_4("BNC_4", 3);
+
+    private final String name;
+    private final int value;
+
+    /**
+     * VideoInputDropdown instantiation
+     *
+     * @param name {@code {@link #name}}
+     * @param value {@code {@link #value}}
+     */
+    VideoInputDropdown(String name, int value) {
+        this.name = name;
+        this.value = value;
+    }
+
+    /**
+     * Retrieves {@code {@link #name}}
+     *
+     * @return value of {@link #name}
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Retrieves {@code {@link #value}}
+     *
+     * @return value of {@link #value}
+     */
+    public int getValue() {
+        return value;
+    }
+
+    /**
+     * Retrieves all name of VideoInputDropdown
+     *
+     * @return list name of Input Interface
+     */
+    public static String[] names() {
```

Comment:
this function is being repeated multiple times throughout other classes. Maybe it would make sense to make a base class for all the "dropdownlist" classes, that would provide this functionality once?